server/Makefile

```diff
@@ -55,33 +54,43 @@ binlinks = \
 	pbench-unpack-tarballs\
 	pbench-verify-backup-tarballs\
 
+serverlib = \
+	setup.py \
+	setup.cfg \
+	requirements.txt \
+	README.md
+
 # targets
 .PHONY: install \
 	install-dirs \
 	install-bin install-binbase install-bintools install-binscripts \
 	install-lib \
-	install-configtools \
+	install-server-lib \
+	install-makefile \
+	install-pbench-scripts \
 	install-build-artifacts
 
-install: install-dirs install-bin install-lib
+install-all: install install-server-lib
+
+install: install-dirs install-bin install-lib install-makefile install-pbench-scripts
 	${COPY} VERSION ${DESTDIR}
 
 install-dirs:
 	${INSTALL} ${INSTALLOPTS} ${DESTDIR}
 	${INSTALL} ${INSTALLOPTS} ${BINDIR}
 	${INSTALL} ${INSTALLOPTS} ${LIBDIR}
 	${INSTALL} ${INSTALLOPTS} ${LIBDIR}/crontab
-	${INSTALL} ${INSTALLOPTS} ${LIBDIR}/configtools
+	${INSTALL} ${INSTALLOPTS} ${LIBDIR}/pbench
 
-install-bin: install-binbase install-bintools install-binscripts install-configtools
+install-bin: install-binbase install-bintools install-binscripts
 
 install-binbase:
 	for f in ${binbase} ;do \
 	    ${COPY} bin/$$f ${BINDIR} \
 	;done
 
 install-bintools:
-	for f in ${bintools} ;do \
+	-for f in ${bintools} ;do \
```

Comment:
Why ignore success/failure here?

Reply:
`pbench-config` and `pbench-server` may not exist: they are created by the setup. I can split them off into a separate target.

Reply:
Actually, that may be a remnant of an aborted change: let me check.

Reply:
Yup, it was a remnant: fixed.